src/config/rcxml.c

```diff
@@ -1036,6 +1036,16 @@ entry(xmlNode *node, char *nodename, char *content)
 		}
 	} else if (!strcasecmp(nodename, "ignoreButtonReleasePeriod.menu")) {
 		rc.menu_ignore_button_release_period = atoi(content);
+	} else if (!strcasecmp(nodename, "width.magnifier")) {
+		rc.mag_width = atoi(content);
+	} else if (!strcasecmp(nodename, "height.magnifier")) {
+		rc.mag_height = atoi(content);
+	} else if (!strcasecmp(nodename, "initScale.magnifier")) {
+		rc.mag_scale = atoi(content);
```

Comment:
Just noticed that this might be set to `0` by the user.
This could then cause divisions by zero in magnify(), so may we should make sure that this is always > 0 in `post_processing()`.

Reply:
To take the final question, no, I think 1.0 + increment is correct rather than starting at the default zoom. My intention was that default zoom is more useful when toggling magnifier on and off. Zooming feels more like something where you just turn the scroll wheel to wherever looks right, so starting at minimum feels right to me.

But yes, good point that the init scale can indeed be a fractional value, so float is more consistent with the increment being a float - will change that. And will also check for zeros - good spot!

Reply:
> To take the final question, no, I think 1.0 + increment is correct rather than starting at the default zoom.

Yes I think you are right, feels more intuitive.

The only thing missing apart from the 0-check and s/int/float/ would then be squashing. Do you want to do that or should I do so when merging?

Reply:
OK, that should all be done, I think.

Do please do the squash when you merge - probably easier that way. 

Many thanks again for all the help and feedback!

Reply:
Merged :)

Reply:
Cool! Many thanks again!